### DISCUSSION TITLE
media mod and Hero11black usb-c connection failed

### DISCUSSION THREAD

**Discussion by Hubsi4** [gopro/labs#532] [Q&A]
Hi everybody, I connected the media mod to my Hero11 using a fully-fledged USB-C extension cable. Don't ask me why, I have a special application. Unfortunately, only charging works, but HDMI out doesn't work. Why? Is the connection between the camera and media mod not a standard USB-C connection? Is there a solution for this?

**Comment by dnewman-gpsw** [comment]
No idea. This is not really a question about Labs, but I will leave it for some Labs user might know.
